docs/rating/rating.html

```diff
@@ -0,0 +1,185 @@
+            >
+          </li>
+        </ul>
+      </nav>
+      <div class="menu-div">
+        <button class="btn btn-primary btn-float" id="menu-btn">
+          <i class="fas fa-bars"></i>
+        </button>
+      </div>
+      <div class="content">
+        <div class="side-nav" id="menu">
+          <div class="side-nav-link">
+            <h2 class="list-heading">Get Started</h2>
+            <button class="btn btn-icon" id="menu-close">
+              <i class="fas fa-times"></i>
+            </button>
+          </div>
+
+          <ul class="list-components">
+            <li class="active">
+              <a href="../get-started.html">Installation</a>
+            </li>
+          </ul>
+          <h2 class="list-heading">Components</h2>
+          <ul class="list-components">
+            <li><a href="../alert/alert.html">Alert</a></li>
+            <li>
+              <a href="../avatar/avatar.html">Avatar</a>
+            </li>
+            <li><a href="../badges/badge.html">Badges</a></li>
+            <li>
+              <a href="../buttons/button.html">Buttons</a>
+            </li>
+            <li><a href="../cards/card.html">Cards</a></li>
+            <li><a href="../images/image.html">Images</a></li>
+            <li><a href="../inputs/input.html">Input</a></li>
+            <li><a href="../navbar/navbar.html">Navbar</a></li>
+            <li><a href="../lists/list.html">List</a></li>
+            <li><a href="../rating/rating.html">Rating</a></li>
+            <li><a href="../toast/toast.html">Toast</a></li>
+            <li><a href="../typography/typography.html">Typography</a></li>
+          </ul>
+        </div>
+        <!-- Main content -->
+        <div class="component-container">
+          <h1 class="component-title">Rating</h1>
+          <div class="component-subHeading"></div>
+
+          <!-- Components -->
+          <div class="component-content">
+            <div class="component-heading">Star Rating</div>
+            <!-- Rating -->
+            <div class="example-live">
+              <div class="rating">
+                <i class="rating-icon fas fa-star filled-star"></i>
+                <i class="rating-icon fas fa-star filled-star"></i>
+                <i class="rating-icon fas fa-star filled-star"></i>
+                <i class="rating-icon fas fa-star filled-star"></i>
+                <i class="rating-icon fas fa-star filled-star"></i>
+              </div>
+            </div>
+          </div>
+          <div class="example-code">
+            <pre>
+              <code class="html hljs xml">
+                <div class="rating">
+                  <i class="rating-icon fas fa-star filled-star"></i>
+                  <i class="rating-icon fas fa-star filled-star"></i>
+                  <i class="rating-icon fas fa-star filled-star"></i>
+                  <i class="rating-icon fas fa-star filled-star"></i>
+                  <i class="rating-icon fas fa-star filled-star"></i>
```

Comment:
the filled class could be used for icons other than the `star icons` as well (ex- the heart icon to add items to wishlist) , so give it a general name, like `filled-icon`

Reply:
Have made the necessary changes